packages/trader/src/Services/perfomance-checker.js

```diff
@@ -0,0 +1,78 @@
+import {
+    isFirefox,
+    isSafari,
+    isEdge,
+} from '_common/browser_detect';
+
+const performanceMetrics = {
+    start               : 'started',
+    firstContentfulPaint: 'first-contentful-paint',
+    smartChartMounted   : 'smart-charts-mounted',
+    purchaseStart       : 'purchase-started',
+    purchaseEnd         : 'purchase-ended',
+    purchaseEnabled     : 'purchase-enabled',
+    tradeEngineStart    : 'trade-engine-started',
+    tradeEngineReady    : 'trade-engine-enabled',
+};
+
+let is_data_sent = false;
+
+function measurePerformance(pushLoadPerformance) {
```

Comment:
can we change `pushLoadPerformance` to `callback`?

Reply:
It's a callback right now